<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialognet</class>
 <widget class="QDialog" name="Dialognet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>تحميل كتب2</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>رابط قائمة الكتب</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxUrls">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonGetList">
           <property name="toolTip">
            <string>تحديث قائمة الكتب</string>
           </property>
           <property name="whatsThis">
            <string>تحديث قائمة الكتب ان كان هناك تغيير في قائمة الكتب</string>
           </property>
           <property name="text">
            <string>تحديث</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ابحث عن كتاب</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonExpande">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../qbooks/ui/images.qrc">
             <normaloff>:/images/image/btnfolder.png</normaloff>:/images/image/btnfolder.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonColapce">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../qbooks/ui/images.qrc">
             <normaloff>:/images/image/colapse.png</normaloff>:/images/image/colapse.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <attribute name="headerDefaultSectionSize">
          <number>250</number>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">الكتاب</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>المؤلف</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="textEditCmd"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditUrl">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>253</green>
           <blue>211</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>232</green>
           <blue>192</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>236</red>
           <green>233</green>
           <blue>216</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>حدد كتابا من القائمة</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qbooks/ui/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialognet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonColapce</sender>
   <signal>clicked()</signal>
   <receiver>treeWidget</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonExpande</sender>
   <signal>clicked()</signal>
   <receiver>treeWidget</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
